5.0/en/0x11-V2-Authentication.md

```diff
@@ -59,13 +59,14 @@ NIST considers SMS as ["restricted" authenticator types](https://pages.nist.gov/
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
 | **2.2.1** | Verify that anti-automation controls are effective at mitigating breached credential testing, brute force, and account lockout attacks. Such controls include blocking the most common breached passwords, soft lockouts, rate limiting, CAPTCHA, ever increasing delays between attempts, IP address restrictions, or risk-based restrictions such as location, first login on a device, recent attempts to unlock the account, or similar. Verify that no more than 100 failed attempts per hour is possible on a single account. | ✓ | ✓ | ✓ | 307 | 5.2.2 / 5.1.1.2 / 5.1.4.2 / 5.1.5.2 |
 | **2.2.2** | [MODIFIED] Verify that weak authenticators (such as out-of-band authentication using VOIP or email) are not used as authentication methods. Verify that restricted authenticators (those using PSTN to deliver OTPs via phone or SMS) are offered only when alternate stronger methods are also offered and when the service provides information on their security risks to users. | ✓ | ✓ | ✓ | 304 | 5.2.10 |
-| **2.2.3** | Verify that secure notifications are sent to users after updates to authentication details, such as credential resets, email or address changes, logging in from unknown or risky locations. The use of push notifications - rather than SMS or email - is preferred, but in the absence of push notifications, SMS or email is acceptable as long as no sensitive information is disclosed in the notification. | ✓ | ✓ | ✓ | 620 | |
+| **2.2.3** | [MODIFIED] Verify that users are notified after updates to authentication details, such as credential resets or modification of the username or email address. | ✓ | ✓ | ✓ | 778 | 6.1.2 |
```

Comment:
I made some changes to both requirements including adding [CWE 778](https://cwe.mitre.org/data/definitions/778.html) and rewording 2.2.10. Are you comfortable with these changes?

Reply:
CWE 778 is really a stretch, since that is about logging and not user notifications.

Reply:
@Sjord can you give me an edit that illustrates what you want to see here? I think we are close to merging this. I agree making the requirement less wordy would be prudent.

Reply:
I propose:

> Verify that users are notified of suspicious authentication attempts. Suspicious authentication attempts include attempts from an unusual location or client, after a long period of inactivity, and partially successful authentication with only one of multiple factors.

I left out "successful authentication after several unsuccessful attempts". I think users should be notified when their account is blocked because of too many login attempts, but perhaps that should not be in this requirement. I don't think this notification should only be done after successful authentication.

Reply:
@jmanico any feedback on @Sjord's response?